litmus-portal/frontend/src/components/Header/ProfileDropdownSection.tsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Box, IconButton } from '@material-ui/core';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreTwoToneIcon from '@material-ui/icons/ExpandMoreTwoTone';
+import React, { useRef, useState } from 'react';
+import ProfileInfoDropdownItems from './ProfileDropdownItems';
+import useStyles from './styles';
+import { Project, ProjectsCallBackType } from '../../models/header';
+
+interface ProfileInfoDropdownSectionProps {
+  name: string;
+  email: string;
+  username: string;
+  projects: Project[];
+  selectedProjectID: string;
+  CallbackToSetSelectedProjectID: ProjectsCallBackType;
+}
+
+const ProfileDropdownSection = (props: ProfileInfoDropdownSectionProps) => {
```

Comment:
It'd be better if you could follow the TS standards like React.FC<> here as well